composer.json

```diff
@@ -28,7 +28,7 @@
 		"wp-coding-standards/wpcs": "2.*",
 		"wp-phpunit/wp-phpunit": "~6.0",
 		"wporg/wporg-repo-tools": "dev-trunk",
-		"yoast/phpunit-polyfills": "^1.0"
+		"yoast/phpunit-polyfills": "^1.1"
```

Comment:
The [PHP unit tests failed](https://github.com/WordPress/wporg-mu-plugins/actions/runs/5932617325) with `Error: Version mismatch detected for the PHPUnit Polyfills. Please ensure that PHPUnit Polyfills 1.1.0 or higher is loaded. Found version: 1.0.5` so I've bumped inline with that

Reply:
Seems like a very new release https://github.com/Yoast/PHPUnit-Polyfills/releases